src/agent/aksLoader.ts

```diff
@@ -50,6 +52,44 @@ export class AKSLoader extends AgentLoader {
                     }
                 )
             );
+
+            // Create metricReaders array and add OTLP reader if environment variables request it
+            try {
+                const metricReaders: MetricReader[] = [];
+                if (
+                    process.env.OTEL_METRICS_EXPORTER === "otlp" &&
+                    (process.env.OTEL_EXPORTER_OTLP_ENDPOINT || process.env.OTEL_EXPORTER_OTLP_METRICS_ENDPOINT)
+                ) {
+                    try {
+                        const otlpExporter = new OTLPMetricExporter({
+                            url: process.env.OTEL_EXPORTER_OTLP_ENDPOINT || process.env.OTEL_EXPORTER_OTLP_METRICS_ENDPOINT,
```

Comment:
I would assume the exporter will use the env variables so no need to explicitly add them here?